open-metadata-implementation/user-interfaces/presentation-server/nodejs/src/client/components/TypeExplorer/README.md

```diff
@@ -0,0 +1,76 @@
+## Type Explorer User Interface
+
+The Type Explorer (Tex) interface provides a way to explore the Egeria type system. It can be used to retrieve the types supported by a chosen server. The type information is then displayed both textually (on the left side) and diagrammatically (on the right side) with a choice of diagrams. One of the diagrams shows the entity inheritance tree and the other diagram shows the types of relationships that an entity type supports.
+
+![Tex Interface](image1)
+
+### Deployment
+The Tex interface runs under the Presentation Server component of Egeria, and it uses the Type Explorer (Tex) View Service in order to retrieve the information you ask for. For details of the Presentation Server and View Server please refer to their readme information.
+
+
+### Configuration
+The Tex View Service is an Integration View Service and is configured using `IntegrationViewServiceConfig`. This contains the name and status of the View Service and also contains a list of the resources that the Tex interface is permitted to access. These resources are the platforms and servers that the user will be able access. The configured servers will appear in the selectors in the Tex interface. All requests to the Tex View Service REST API are based on these configured named resources. When a user selects a server name from the selector lists, the interface sends the resource name to the Tex View Service, which resolves the server name to a resource endpoint to identify the URL needed to send a request to the server. 
```

Comment:
I suggest for the first sentence "The Tex View Service is an Integration View Service and is configured using admin API command (link); the configuraiton is supplied in `IntegrationViewServiceConfig` (link) ".

Second sentence  I am not sure what status is of the View Service . I suggest detailing what that means for the configuration. 
Third sentence - missing to before access.
6th sentence I did not quite understand "When a user selects a server name from the selector lists, the interface sends the resource name to the Tex". Is the server name a resource name? I am not sure I understand what a resourse is here.

Reply:
OK - thanks again. Have modified the first part as per your recommendation.

The part about status is referring to "viewServiceOperationalStatus":"ENABLED",  in the config.

Have made corrections and expanded for all the points you raised here. Hopefully the result is OK.